.cirrus/cirrus-promote-webapp-assets.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ "$PULL_REQUEST" != "" ]; then
+  PRIVATE_TARGET_REPO='sonarsource-private-dev'
+  PUBLIC_TARGET_REPO='sonarsource-public-dev'
+  STATUS='it-passed-pr'
+# CirrusCI build on master branch
+elif [[ "$CIRRUS_CI" == "true" ]] && [[ ("$GITHUB_BRANCH" == "master" || "$GITHUB_BRANCH" == "branch-"*) ]]; then
+  PRIVATE_TARGET_REPO='sonarsource-private-builds'
+  PUBLIC_TARGET_REPO='sonarsource-public-builds'
+  STATUS='it-passed'
+fi
+
+if [ -n "${STATUS:-}" ]; then
+  echo "Promoting build sonarqube-webapp-assets#$BUILD_NUMBER"
```

Comment:
NP maybe the string "sonarqube-webapp-assets" could be stored in a variable as it's repeated multiple times?

Reply:
can you move that comment to the other repo, this PR is just there for validation and won't be merged.